assets/themes/zeppelin/css/style.css

```diff
@@ -1134,7 +1195,225 @@ and (max-width: 1024px) {
   .carousel-indicators {
     margin-bottom: -60px;
   }
+
   .carousel-caption {
     padding-bottom: 60px;
   }
 }
+
+/* Advertisement Section */
+
+.advertise {
+  background: #FFF;
+  padding-bottom: 15px;
```

Comment:
3. more `padding-bottom` for `.advertise`. It's too small compared to `padding-top`. 

`padding-bottom: 35px;` would be proper.

#### Before

![image](https://cloud.githubusercontent.com/assets/4968473/26775363/995a626c-4a0f-11e7-9086-898cf766cc18.png)

#### After

![image](https://cloud.githubusercontent.com/assets/4968473/26775387/af338ba4-4a0f-11e7-948e-81b95ff8f480.png)